terraform/core-infra/azure/network.tf

```diff
@@ -0,0 +1,28 @@
+data "azurerm_virtual_network" "plural" {
```

Comment:
we should be creating a dev + prod subnet here, not importing them. Check the implementations for aws/gcp to see the pattern.

The service context for the management cluster should be entirely created within its own terraform stack.